code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -496,6 +511,12 @@
 			return 0
 	return 1
 
+/mob/living/simple_animal/proc/attack_threshold_check(damage, damagetype = BRUTE)
+	if(damage <= force_threshold || ignored_damage_types[damagetype])
+		visible_message("<span class='warning'>[src] looks unharmed.</span>")
+	else
+		adjustBruteLoss(damage)

```

Comment:
@TheDZD i see what you mean, should i do it as an if damagetype == [type] check? and remove the ignore_damage_type from the adjust calls?